src/migrant/tests/test_repo.py

```diff
@@ -50,9 +51,10 @@ def test_weird_list(self):
         ====
         1babe_third.py
         >>>> CONFLICT MARKER
```

Comment:
uh... is that a merge conflict that got merged?

Reply:
Does look like one!

Reply:
```
a2016b65 (Andrey Lebedev  2014-05-02 17:29:06 +0300 49)         <<<< HEAD
a2016b65 (Andrey Lebedev  2014-05-02 17:29:06 +0300 50)         d724a_second.py
a2016b65 (Andrey Lebedev  2014-05-02 17:29:06 +0300 51)         ====
a2016b65 (Andrey Lebedev  2014-05-02 17:29:06 +0300 52)         1babe_third.py
a2016b65 (Andrey Lebedev  2014-05-02 17:29:06 +0300 53)         >>>> CONFLICT MARKER
```

Reply:
Gonna triple check it's not me though.

Reply:
Ah, that's a deliberate to test survive conflict merges.

Reply:
Not what I'd want (fail!) but it's already there.